DataFormats/Scouting/test/testDataFormatsScoutingRun2.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Pass in name and status
+function die {
+  printf "\n%s: status %s\n" "$1" "$2"
+  if [ $# -gt 2 ]; then
+    printf "%s\n" "=== Log File =========="
+    cat $3
+    printf "%s\n" "=== End of Log File ==="
+  fi
+  exit $2
+}
+
+# run test job
+TESTDIR="${LOCALTOP}"/src/DataFormats/Scouting/test
+
+cmsRun "${TESTDIR}"/scoutingCollectionsIO_cfg.py -- \
```

Comment:
These could now be just
```suggestion
cmsRun "${SCRAM_TEST_PATH}"/scoutingCollectionsIO_cfg.py -- \
```